onboard/nav/stateMachine.hpp

```diff
@@ -112,6 +116,12 @@ class StateMachine
     // Bool of whether radio repeater has been dropped.
     bool mRepeaterDropComplete = false;
```

Comment:
I fixed this in one of my commits so maybe rebasing will fix this but initialize this in the constructor rather than the hpp file.